test/scss/_feature-targeting-test.scss

```diff
@@ -80,6 +81,13 @@
     // Form Field
     @include mdc-form-field-core-styles($query: $query);
 
+    // Image List
+    @include mdc-image-list-core-styles($query: $query);
+    @include mdc-image-list-aspect(1, $query: $query);
+    @include mdc-image-list-shape-radius(0, $rtl-reflexive: false, $query: $query);
+    @include mdc-image-list-standard-columns(4, $gutter-size: $mdc-image-list-standard-gutter-size, $query: $query);
```

Comment:
Add missing import

```
@import "@material/image-list/variables";
```

The reason it is working because I think the mixin internally imports `_variable.scss`.

Reply:
Oh, whoops...actually I meant to replace those variables with dummy values. Clearly I sent this one out too fast or my attention got interrupted.